packages/babel-plugin-relay/createModernNode.js

```diff
@@ -125,12 +125,12 @@ function getRelativeImportPath(
   const relative = path.relative(
     path.dirname(filename),
     path.resolve(artifactDirectory),
-  );
+  ).replace(/\\/g, '/');
 
   const relativeReference =
     relative.length === 0 || !relative.startsWith('.') ? './' : '';
```

Comment:
This line does not use the operating system's delimiter, so we are mixing forward slashes and backslashes on windows

Reply:
Yeah..my mistake...yes this regex is fine.. 👍